app/views/user_mailer/_work_info.text.erb

```diff
@@ -0,0 +1,12 @@
+<% # expects work %>
+<%= work_metadata_label(Work.human_attribute_name("chapter_total_display"))%><%= work.chapter_total_display %>
+<%= work_tag_metadata(work.fandoms) %>
+<%= work_tag_metadata(work.ratings) %>
+<%= work_tag_metadata(work.archive_warnings) %><% unless work.relationship_string.blank? %>
```

Comment:
(It's like this so we don't end up with extra blank lines in the email when a tag type is not present.)

Reply:
Counterintuitively enough, I think you can actually prevent extra blank lines in the output by adding newlines. A template like this produces three blank lines:
```
<% if false %><%= "Text." %><% end %>
<% if false %><%= "Text." %><% end %>
<% if false %><%= "Text." %><% end %>
```
Whereas a template like this produces no blank lines:
```
<% if false %>
<%= "Text." %>
<% end %>
<% if false %>
<%= "Text." %>
<% end %>
<% if false %>
<%= "Text." %>
<% end %>
```
(This is because a `<% %>` tag that's alone on the line doesn't affect the output. But multiple tags in one line are trickier.)

Reply:
Oh, that would be much better -- thanks!